src/main/webapp/app/exercises/text/manage/example-text-submission/example-text-submission.component.ts

```diff
@@ -57,7 +57,8 @@ export class ExampleTextSubmissionComponent extends TextAssessmentBaseComponent
     SubmissionButtonStates = SubmissionButtonStates;
     AssessButtonStates = AssessButtonStates;
     UIStates = UIStates;
-    selectedMode: 'readConfirm' | 'assessCorrectly';
+    selectedMode: ExampleSubmissionMode;
+    ExampleSubmissionMode = ExampleSubmissionMode;
```

Comment:
We often have such variables separated from the rest and preceded with `readonly`